sphinx/themes/agogo/static/rtl.css.jinja

```diff
@@ -0,0 +1,156 @@
+{% if theme_is_rtl|tobool %}
+
+/* Core RTL overrides for Agogo theme */
+body,
+div.header div.rel {
+    direction: rtl !important;
+}
+
+body {
+    text-align: right !important;
+}
+
+/* Header adjustments */
+div.header .headertitle {
+    text-align: right !important;
+    letter-spacing: 0 !important;
+}
+
+p.logo {
+    float: left !important;
```

Comment:
This really shouldn’t be needed. Just change the `agogo.css.jinja`’s definition to use flow-relative values: 

```css
p.logo {
    float: inline-start;
}
```